test/mock.js

```diff
@@ -0,0 +1,711 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const Bad_Request = {
+    err: {
+    throws: new Error('Bad Request')
+    },
+    message: "Bad Request"
+
+}
+const Unauthorized_Request = {
+    err: {
+    throws: new Error('Unauthorized')
+    },
+    message: "Unauthorized"
+}
+const Forbidden_Request = {
+    err: {
+    throws: new Error('Forbidden Request')
+    },
+    message: "Forbidden Request"
+}
+const Not_Found = {
+    err: {
+    throws: new Error('Not Found')
+    },
+    message: "Not Found"
+}
+const Internal_Server_Error = {
+    err: {
+    throws: new Error('Internal Server Error')
+    },
+    message: "Internal Server Error"
+}
+
+const activities = {
+    "total": 2,
+    "offset": 0,
+    "limit": 10,
+    "activities": [
+        {
+            "id": 168805,
+            "thirdPartyId": "2fc846c4-d5c6-4d67-9ef8-939bb907cc71",
```

Comment:
Since we are mocking the fetch anyway, is it possible to put a dummy ID here just to be safe?

Reply:
This is taken from public example from Target API docs - https://developers.adobetarget.com/api/#list-activities

Reply:
aah then its cool